airflow/www/views.py

```diff
@@ -1084,6 +1084,28 @@ def blocked(self):
             })
         return wwwutils.json_response(payload)
 
+    @expose('/latest_dag_runs')
+    @login_required
+    def latest_dag_runs(self):
+        session = settings.Session()
+        DR = models.DagRun
+        subquery = (
+            session.query(DR.dag_id, sqla.func.max(DR.execution_date).label('execution_date'))
+            .filter(DR.state == State.RUNNING)
+            .group_by(DR.dag_id)
+            .subquery()
+        )
+        dagruns = session.query(DR).join(subquery, and_(DR.dag_id == subquery.c.dag_id, DR.execution_date == subquery.c.execution_date)).all()
+        payload = []
+        for dagrun in dagruns:
```

Comment:
Let's be defensive and shield against dagruns that don't have these properties defined.